spec/features/webauthn/sign_in_spec.rb

```diff
@@ -49,4 +49,19 @@
     expect(page).to have_content(t('errors.general'))
     expect(page).to have_current_path(login_two_factor_webauthn_path)
   end
+
+  it 'does not show error after successful challlenge/secret reattempt' do
```

Comment:
Minor: Typo on the "challenge"

```suggestion
  it 'does not show error after successful challenge/secret reattempt' do
```